internal/pkg/cli/pipeline_update.go

```diff
@@ -239,6 +242,16 @@ func (o *updatePipelineOpts) Execute() error {
 	}
 	o.pipelineName = pipeline.Name
 
+	// If the source has an existing connection, get the correlating ConnectionARN .
+	connection, ok := pipeline.Source.Properties["connection_name"]
+	if ok {
+		arn, err := o.codestar.GetConnectionARN((connection).(string))
+		if err != nil {
+			return fmt.Errorf("get connection ARN: %w", err)
+		}
+		pipeline.Source.Properties["connection_arn"] = arn
```

Comment:
A question: are we adding `arn` to the manifest object so that it's passed to `PipelineSourceFromManifest` or are `connection_arn` also part of the manifest?

Reply:
The former-- we're adding it here in the cli package to pass to the deploy package, yup.